File: uilib/src/main/java/dev/entao/util/app/SystemProperties.java
package dev.entao.util.app;


import dev.entao.util.RefUtil;

public class SystemProperties {
	private static final String CLS = "android.os.SystemProperties";

	public static String get(String key) {
		return (String) RefUtil.invokeStatic(CLS, "get", key);
	}

	public static String get(String key, String defVal) {
		return (String) RefUtil.invokeStatic(CLS, "get", new Class<?>[]{String.class , String.class}, key, defVal);
	}
}

File: uilib/src/main/java/dev/entao/util/app/SystemProperties.kt
package dev.entao.util.app


import dev.entao.util.RefUtil

object SystemProperties {
    private val CLS = "android.os.SystemProperties"

    operator fun get(key: String): String {
        return RefUtil.invokeStatic(CLS, "get", key) as String
    }

    operator fun get(key: String, defVal: String): String {
        return RefUtil.invokeStatic(
            CLS,
            "get",
            arrayOf<Class<*>>(String::class.java, String::class.java),
            key,
            defVal
        ) as String
    }
}
